.github/workflows/scripts/compare_translations.py

```
@@ -0,0 +1,236 @@
+                print(f"File {language}/{file} has missing translations for:")
+                for error in errors:
+                    print(f"  - {error}")
+
+    if error_found:
+        sys.exit(1)  # Exit with an error status code
+    else:
+        print("All translations are present")
+        sys.exit(0)
+
+
+def main():
```

Ruff (0.12.2)

Missing return type annotation for public function `main`

Add return type annotation: `None`

(ANN201)